### PR TITLE
HPCC-17084 New LZ Structure

### PR DESCRIPTION
**Pull request by miguelvazq**
Restructure ECLWatch to allow for the new Landing zone structure to be displayed.

Signed-off-by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review

File: esp/src/eclwatch/LZBrowseWidget.js [hunk 1]
```diff
@@ -630,7 +663,10 @@ define([
                             var name = _name;
                             if (row.isDir === undefined) {
                                 img = dojoConfig.getImageHTML("server.png");
-                                name += " [//" + row.NetAddress + row.Path + "]";
+                                name += " [" + row.Path + "]";
+                            } else if (row.isMachine) {
+                                 img = dojoConfig.getImageHTML("machine.png");
+                                 name;
```
Comment (GordonSmith) [inline esp/src/eclwatch/LZBrowseWidget.js]: This line does nothing.

File: esp/src/eclwatch/LZBrowseWidget.js [hunk 2]
```diff
@@ -177,19 +186,35 @@ define([
...
+            }
+
+            if (!this.dropZoneMachineSelect.initalized) {
+                this.dropZoneMachineSelect.init({
+                    DropZoneMachines: true,
+                    callback: function (value, row) {
+                        targetRow.machine.Name = value
+                        targetRow.machine.Netaddress = value
                         if (context.dropZoneFolderSelect) {
-                            context.dropZoneFolderSelect.reset();
```
Comment (GordonSmith) [inline esp/src/eclwatch/LZBrowseWidget.js]: Is the reset no longer needed?  (was it ever?)

**Review by GordonSmith — CHANGES_REQUESTED** [review]

File: esp/src/eclwatch/LZBrowseWidget.js (hunk 2 above)
Comment (miguelvazq) [inline esp/src/eclwatch/LZBrowseWidget.js]: After various testing, the .reset() did not actually clear out the values on the drop down. So, I do not believe it was needed.

**Comment by miguelvazq** [comment]
@GordonSmith updated.

**Review by GordonSmith — APPROVED** [review]
Looks fine.

**Pull request merged**